src/js/snapshot.js

```diff
@@ -155,6 +251,10 @@ snapshot.prototype.checkAll = function (cBox) {
 	objOdata.checkBoxSelectOData(cBox, buttonId);
 	objCommon.showSelectedRow(document.getElementById("snapshotchkSelectall"),"row","wdrowid");
 	var noOfRecords = $("#snapshotTable > tbody > tr").length;
+	if ($("#nowExportBar").length) {
```

Comment:
Add a dummy class to do the counting.
Example:  

       $('#snapshotTable .exportedRow').hide()